### DISCUSSION TITLE
Old versions documentation access

### DISCUSSION THREAD

**Discussion by HenBK** [pocketbase/pocketbase#5966] [Q&A] [answered]
Hi there, I noticed that the documentation on [https://pocketbase.io/docs/ ](https://pocketbase.io/docs/ ) got updated since the v0.23.X release, I wonder if there is any way of consulting the 0.22.X version documentation, I'm planning to port and upgrade versions but I don't think I will do it any time soon, so I would like to know if there is any way of getting to the documentation of the version I'm currently using (v0.22.X) just to be able to easily add new features that I'll port later when I stop postponing the upgrade. Cheers!

**Comment by chnfyi** [comment, marked as answer]
Hi, you can consult the <=0.22.x documentation [here](https://pocketbase.io/old/docs/). Make sure to consult the changelog too before starting to work on your upgrade.

**Reply by HenBK** [reply]
Thanks a lot!
